src/transformers/models/detr/modeling_detr.py

```diff
@@ -2288,6 +2284,17 @@ def generalized_box_iou(boxes1, boxes2):
     return iou - (area - union) / area
 
 
+# Copied from transformers.models.detr.feature_extraction_detr.center_to_corners_format
+def center_to_corners_format(x):
+    """
+    Converts a PyTorch tensor of bounding boxes of center format (center_x, center_y, width, height) to corners format
+    (x_0, y_0, x_1, y_1).
+    """
+    x_c, y_c, w, h = x.unbind(-1)
```

Comment:
As mentioned above, let's also change `feature_extraction_detr.center_to_corners_format` to upgrade the variable names :-)